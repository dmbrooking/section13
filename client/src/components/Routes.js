import React from 'react'
import { Route } from 'react-router-dom'
import App from './App'
import RequireAuth from './auth/RequireAuth'
import LoginContainer from '../containers/LoginContainer'
import Home from './Home'

const Routes = () => {
  return (
    <App>
      <Route exact path='/' component={RequireAuth(Home)} />
      <Route exact path='/login' component={LoginContainer} />
    </App>
  )
}

export default Routes
