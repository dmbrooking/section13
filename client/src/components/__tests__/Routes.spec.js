import React from 'react'

import Routes from '../Routes'

describe('<Routes />', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Routes />)
  })

  it('matches saved snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
