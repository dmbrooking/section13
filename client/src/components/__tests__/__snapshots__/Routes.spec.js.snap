// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> matches saved snapshot 1`] = `
<App>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
</App>
`;
